providers/shared/components/component.ftl

```diff
@@ -131,165 +131,6 @@
 [#assign USERPOOL_AUTHPROVIDER_COMPONENT_TYPE = "userpoolauthprovider" ]
 [#assign USERPOOL_RESOURCE_COMPONENT_TYPE = "userpoolresource" ]
 
-[#assign
-    filterChildrenConfiguration = [
```

Comment:
A filter is basically what is now being referred to as a district. A link is basically a district with some extra attributes, which is why there were two separate definitions here.

It doesn't need addressing now but at some point, the link attribute set may need to be separated into two structure again.

Reply:
We can do that easily enough when the time comes, no?